paddle/fluid/pybind/imperative.cc

```diff
@@ -424,7 +425,15 @@ static void ParseIndexingSlice(
   // We allow indexing by Integers, Slices, Ellipsis, None, tuples of those
   // types, and list of Bool and Integers.
   // wrap to tuple
+
+  // NOTE(zhiqiu): PyTuple_Pack increases refcount.
   PyObject *index = !PyTuple_Check(_index) ? PyTuple_Pack(1, _index) : _index;
+  DEFINE_PADDLE_SCOPE_GUARD([&index, &_index]() {
```

Comment:
NIT. You can use `[index, _index]` here. Since `index` and `_index` are pointers.

Reply:
done, thx.